out/blocks/wd_admin_payment_main_form.tpl

```diff
@@ -119,6 +119,57 @@
         [{oxinputhelp ident="config_three_d_merchant_secret_desc"}]
     </td>
   </tr>
+
```

Comment:
Can u extend the base template and create a new creditcard template

Reply:
This is addressed in Pull Request #32 . Let's prioritise that one and keep this open until #32 is merged.